pkg/controller/infrastructure/configvalidator_test.go

```diff
@@ -186,6 +187,122 @@ var _ = Describe("ConfigValidator", func() {
 				"Detail": Equal(fmt.Sprintf("could not get VPC attribute enableDnsSupport for VPC %s: test", vpcID)),
 			}))
 		})
+
+		Describe("validate Elastic IP addresses", func() {
+			BeforeEach(func() {
+				infra.ClusterName = "cluster-1"
+				infra.Spec.ProviderConfig.Raw = encode(&apisaws.InfrastructureConfig{
+					Networks: apisaws.Networks{
+						VPC: apisaws.VPC{},
+						Zones: []apisaws.Zone{
+							{
+								ElasticIPAllocationID: pointer.String("eipalloc-0e2669d4b46150ee4"),
+							},
+							{
+								ElasticIPAllocationID: pointer.String("eipalloc-0e2669d4b46150ee5"),
+							},
+							{
+								ElasticIPAllocationID: pointer.String("eipalloc-0e2669d4b46150ee6"),
+							},
+						},
+					},
+				})
+			})
+
+			It("should succeed - no EIPs configured", func() {
+				infra.Spec.ProviderConfig.Raw = encode(&apisaws.InfrastructureConfig{
+					Networks: apisaws.Networks{
+						VPC: apisaws.VPC{},
+					},
+				})
+				errorList := cv.Validate(ctx, infra)
+				Expect(errorList).To(BeEmpty())
+			})
+
+			It("should succeed - all EIPs exist and are already associated to the Shoot's NAT Gateways", func() {
+				mapping := map[string]*string{
+					"eipalloc-0e2669d4b46150ee4": pointer.String("eipassoc-0f8ff66536587824b"),
```

Comment:
What about using simpler/easier-to-read EIP IDs for the test instead of such long names?